mora/service/orgunit.py

```diff
@@ -146,7 +146,7 @@ def edit(self, unitid, req):
                     addrs,
                     address.get_relation_for(
                         original_data[keys.ADDRESS_TYPE],
-                        original_data[keys.VALUE],
+                        original_data,
                         original_data[keys.VALIDITY],
```

Comment:
Am I tired, or are you sending the validity as the fallback here? Does that make sense?

Reply:
> Am I tired, or are you sending the validity as the fallback here? Does that make sense?

Erm, good question…

Reply:
>> Am I tired, or are you sending the validity as the fallback here? Does that make sense?

> Erm, good question…

The answer appears to be that our tests [do not cover](http://lorajenkins.atlas.magenta.dk/job/mora/view/change-requests/job/PR-85/1/cobertura/service/orgunit_py/) this code, so it couldn't possibly work 😞